### PR TITLE
(Partner ID: cyware-238) - CTIX new V3 app for XSOAR ( CLA compatible copy of PR https://github.com/demisto/content/pull/19060) 

### PR DESCRIPTION
**Pull request by cywaredev**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by cywaredev** [comment]
@JasBeilin @Noy-Maimon @edik24 This is a new PR with the code of https://github.com/demisto/content/pull/19060 as this now resolved the issues I was facing in signing the CLA. Request you to continue the rest of the process here.

**Comment by Noy-Maimon** [comment]
Hi @cywaredev, thanks for the update - please also sign the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA) so we could move forward. Thanks!

File: Packs/CTIX/Integrations/CTIX/README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 This is Cyware Threat Intelligence eXhange(CTIX) integration which enriches IP/Domain/URL/File Data.
 This integration was integrated and tested with version 2.4 and 2.7 of CTIX.
+This integration is NOT COMPATIBLE with CTIX version 3.0 and above. Use the CTIX_V3 Pack for CTIX version 3 and above.
```
Comment (ShahafBenYakir) [inline Packs/CTIX/Integrations/CTIX/README.md:3]:
```suggestion
This integration is NOT COMPATIBLE with CTIX version 3.0 and above. Use the CTIX V3 Integratoin for CTIX version 3 and above. 
```

**Comment by Noy-Maimon** [comment]
As discussed in the demo, here are the issues that are left to fix:
1. DBot score - make sure all relevant command outputs dbot score to the context data. soo more info [here](https://xsoar.pan.dev/docs/integrations/dbot). 
2. Indicators - make sure all relevant commands outputs indicators to the context data. see more info [here](https://xsoar.pan.dev/docs/integrations/context-standards-mandatory).
3. Errors handling - when an error is raised, catch the error and raise an error only with the error message, and present it gracefully to the user. see more info [here](https://xsoar.pan.dev/docs/integrations/code-conventions#exceptions-and-errors).
4. 'Results' context data key - change the key to be something more representing of the command that generated it. see more info [here](https://xsoar.pan.dev/docs/integrations/context-and-outputs).

**Pull request merged**